commands/commands.go

```diff
@@ -52,7 +53,7 @@ type Executor interface {
 
 // Build constructs root command and returns it.
 func Build(e Executor) *cobra.Command {
-	return newRootCmd(e, newSurvey()).cmd
```

Comment:
we should have tests for survey. we already had some tests, why removing them within this commit?